lib/ObjWriter/jitDebugInfo.h

```diff
@@ -0,0 +1,46 @@
+#ifndef JIT_DEBUG_INFO_H
+#define JIT_DEBUG_INFO_H
+
+typedef unsigned int DWORD;
+#define _TARGET_AMD64_ 1
+
```

Comment:
Tried but didn't work. Saw new build failures such as

 C:\Program Files (x86)\Windows Kits\10\Include\10.0.10586.0\shared\minwindef.h(156): error C2371: 'DWORD': redefinition; different basic types [d:\build\llilc-build\lib\ObjWriter\objwrit
er.vcxproj]
  C:\Program Files (x86)\Windows Kits\10\Include\10.0.10586.0\um\winnt.h(2591): error C2733: '_BitScanForward': second C linkage of overloaded function not allowed [d:\build\llilc-build\li
b\ObjWriter\objwriter.vcxproj]
  C:\Program Files (x86)\Windows Kits\10\Include\10.0.10586.0\um\winnt.h(2598): error C2733: '_BitScanReverse': second C linkage of overloaded function not allowed [d:\build\llilc-build\li
b\ObjWriter\objwriter.vcxproj]
  C:\Program Files (x86)\Windows Kits\10\Include\10.0.10586.0\um\winnt.h(2605): error C2733: '_BitScanForward64': second C linkage of overloaded function not allowed [d:\build\llilc-build\
lib\ObjWriter\objwriter.vcxproj]
  C:\Program Files (x86)\Windows Kits\10\Include\10.0.10586.0\um\winnt.h(2612): error C2733: '_BitScanReverse64': second C linkage of overloaded function not allowed [d:\build\llilc-build\